BuildDetails.md

```diff
@@ -0,0 +1,57 @@
+
+
+## Grunt Tasks
+The following ten tasks are run, in order, every time you build the project with grunt. The end result of the build is a new directory named Distribution. This directory will contain all of the project files in minified form.
+
+### clean
+The clean task deletes all files and folders in the project's Distribution directory. This occurs because the Distribution folder must be recreated to reflect any changes made to the files. See [this](https://github.com/gruntjs/grunt-contrib-clean) page for more information on the task.
+
+### tslint
+In order to make sure that the .ts files in the project are neat and readable, grunt runs tslint. This task checks each file for whitespaces, alignment issues, unused variables, missing semicolons, and more, throwing errors if it finds any issues. See [this](https://github.com/palantir/grunt-tslint) page for more information on tslint.
+
+### typescript
+#### typescript:base
+This task translates the project's main .ts file into a .js file. This is necessary because, unlike JavaScript, TypeScript is not compatible with HTML.
+#### typescript:distribution
+The typescript:distribution task translates the project's main .ts file into a .js file that is placed in the Distribution directory and concatenated with the version number of the project.
+
+See [this](https://github.com/k-maru/grunt-typescript) page for more information on typescript.
+
+### copy
+This task copies files and folders from the Source directory into the Distribution directory. 
+* It copies the project's main .ts file into Distribution, concatenating the project version to the name, and does the same to the accompanying .d.ts file. 
+* It also copies the file created by typescript:distribution to the directory in Distribution named after the project name and version. 
+* The References directory in Source is copied to the Distribution directory, too. 
+* The README.md and LICENSE.txt files are both copied into Source and Distribution. 
+* The last things copied into the Distribution directory are the Source/Fonts, Source/Sounds, and Source/Theme directories.
+
+See [this](https://github.com/gruntjs/grunt-contrib-copy) page for more information on the task.
+
+### uglify
+The uglify:dist task minifies the .js files. The files are compressed into two files: One of them contains the file created when typescript:distribution was run along with all .js file in the settings and maps directories while the other file minifies index.js. A sourcemap is created for each of these new files. See [this](https://github.com/gruntjs/grunt-contrib-uglify) page for more information on uglify.
+
+### cssmin
+cssmin:dist essentially does the same thing as uglify:dist but with .css files. It compresses index.css and creates a sourcemap for the newly compressed file. Both of these files can be found in the Distribution directory. See [this](https://github.com/gruntjs/grunt-contrib-cssmin) page for more information on the task.
```

Comment:
You can just say it minifies the .css files. Your perspective might be that .css minification is a less-significant variation of .js minification, but they're just two similar variations on a theme.